src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+
+import { AppContext } from '../AppContext';
+
+export const ErrorMessage: React.FC = React.memo(() => {
```

Comment:
The base filename does not match the name of its primary named export. Consider renaming the file to ErrorMessage.tsx, or change the export to default.